content/en/docs/setup/install/external-controlplane/index.md

```diff
@@ -229,9 +304,6 @@ and installing the sidecar injector webhook configuration on the remote cluster
         rootNamespace: external-istiod
         defaultConfig:
           discoveryAddress: $EXTERNAL_ISTIOD_ADDR:15012
-          proxyMetadata:
```

Comment:
Removing these breaks the "production" instructions.

Reply:
@frankbu this should not break production setup since cacertificates file is the default. I can add it back if you insist.

Reply:
Oh, in that case it sounds like unnecessary noise in the instructions with or without your other changes. I don't know who added it in the first place, but if it works without it we should simplify the config and remove it.

Reply:
right. this should not be needed.